Tiltfile

```diff
@@ -78,7 +105,7 @@ def enable_provider(name):
     # Set up a local_resource build of the provider's manager binary. The provider is expected to have a main.go in
     # manager_build_path. The binary is written to .tiltbuild/manager.
     local_resource(name + "_manager",
-                   cmd='cd ' + context + ';mkdir -p .tiltbuild;CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -ldflags \'-extldflags "-static"\' -o .tiltbuild/manager',
+                   cmd='cd ' + context + ';mkdir -p .tiltbuild;CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -race -ldflags \'-extldflags "-static"\' -o .tiltbuild/manager',
```

Comment:
I'm not against enabling the race detector, but do we want to switch to using cgo? I'm wondering if this would be better as a separate test target vs the default for iterative development.

Reply:
Fair point. Will enable build cache, but leave race off.